gateway/tests/api/test_v1_serializers.py

```diff
@@ -261,9 +261,6 @@ def test_upload_program_serializer_with_only_title(self):
 
     def test_upload_program_serializer_allowed_dependencies(self):
         """Tests dependency allowlist."""
-
-        print("TEST: Program succeeds if all dependencies are allowlisted")
-
```

Comment:
Did you want to remove these? If so, that's ok -- I just added those so I could see which output went with which test (which ideally we'd have for all tests).